app/code/Magento/Wishlist/view/frontend/css/styles.css

```diff
@@ -0,0 +1,13 @@
+.limiter {
```

Comment:
Please move styles to app/code/Magento/Wishlist/view/frontend/web/css/source/_module.less